### PR TITLE
Build and Pushing Docker Image

### PR DESCRIPTION
**Pull request by rohank63**
In this commit **.travis.yml** has been updated with the following changes :

- Building of Docker Image.
- Running of Docker Image.
- Automatic building and pushing of Docker images to Docker Hub (CI pipeline).

However, still the integration between **stage : build** and **stage : test** has to be done, that will be taken care once the other general issues are fixed in the next commits.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,4 +40,16 @@ jobs:
     # integration tests
     - src/htsinfer.py --help
     - src/htsinfer.py --debug
+  
+  - stage: push_image
```
Comment (uniqueg) [inline .travis.yml:44]: This won't work. You need to mention, as I mentioned earlier, that different stages _run in parallel_, on different VMs. So if you have a stage that just pushes a workflow, there will be no image to push. So the code to build the image needs to be added to this stage as well.

File: .travis.yml [hunk 2]
```diff
@@ -8,16 +8,29 @@ services:
...
+
+stages: 
+- name: build_image
 - name: test
+- name: push_image
+  if: branch = master
 
 jobs:
   include:
+  - stage: build_image
```
Comment (uniqueg) [inline .travis.yml:20]: To keep the CI config simple, I think for all intents and purposes, building and running the Docker image is a test, too, so I would add it to the test stage, at least for that.

File: MANIFEST.in [hunk 3]
```diff
@@ -4,3 +4,4 @@
 
 exclude .gitignore
 exclude .gitlab-ci.yaml
+exclude .vscode
```
Comment (uniqueg) [inline MANIFEST.in:7]: `.vscode` should not be under version control (and thus should be specified in `.gitignore`, which I think it is), which means that it doesn't need to be explicitly specified here as long as `setuptools-git` is used in `setup.py` (which I think it is). Check the comment in lines 1-3 for more details.

File: .travis.yml (hunk 1 above)
Comment (rohank63) [reply .travis.yml:44]: Yeah , I know that we discussed yesterday, about the putting them into a single stage, that's why I wrote in the description I will combine these stages after adding the remaining issues-
- after versioning scheme.
- CI pipeline for automatic PyPI publishing.
Nonetheless I will make all the changes as you proposed and do the things accordingly, and take care of all the issues.

File: .travis.yml [hunk 4]
```diff
@@ -8,16 +8,29 @@ services:
 
 if: type != pull_request  # do not build anything for PRs
```
[the remark this answers is not in the document]
Comment (rohank63) [reply .travis.yml:9]: Sure, I will add the condition and do the required changes.

**Pull request closed without merging**